README.md

```diff
@@ -62,6 +62,10 @@ If your Mattermost server is running locally, you can enable [local mode](https:
         ...
         "EnableLocalMode": true,
         "LocalModeSocketLocation": "/var/tmp/mattermost_local.socket"
+    },
+    "PluginSettings" : {
+        ...
+        "EnableUploads" : true
```

Comment:
`EnableUploads` need to be enabled regardless of the use of Local Mode. Could the paragraph in `Development` make it clear, that `EnableUploads` needs to be enabled?